.github/workflows/build.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         run: |
           BRANCH="${GITHUB_REF##*/}"
           CI_TAG=${BRANCH#v}-ci
-          if [ ${BRANCH} = "master" ]; then
+          if [ ${BRANCH} = "main" ]; then
```

Comment:
There is reference [here](https://github.com/openebs/linux-utils/blob/24b0576aa8779594e5eb542cbed7b858f427d5fd/.github/workflows/build.yml#L117) to master. Please change that also.